### PR TITLE
updates of dependencies

### PR DESCRIPTION
**Pull request by aleqi200**
updated vert.x to 3.5.4
removed joda-time for java-time
added jenv .java-version
changed reflections to java 9 compatible deprecated warnings

### CODE REVIEW

File: src/main/java/com/groupon/vertx/utils/HealthcheckHandler.java [hunk 1]
```diff
@@ -35,7 +35,7 @@
     private static final String CONTENT_TYPE = "text/plain";
     private static final String CACHE_CONTROL = "private, no-cache, no-store, must-revalidate";
 
-    public abstract void handle(final HttpServerRequest request);
```
Comment (cchacin) [inline src/main/java/com/groupon/vertx/utils/HealthcheckHandler.java]: unintended formatting changes?

File: src/main/java/com/groupon/vertx/utils/config/ConfigParser.java [hunk 2]
```diff
@@ -32,5 +32,5 @@
      * @param configuration the configuration content
      * @return the JsonObject representing the configuration
      */
-    JsonObject parse(final String configuration);
+    JsonObject parse(String configuration);
```
Comment (cchacin) [inline src/main/java/com/groupon/vertx/utils/config/ConfigParser.java]: unintended formatting changes?

File: src/main/java/com/groupon/vertx/utils/HealthcheckHandler.java (hunk 1 above)
Comment (aleqi200) [reply src/main/java/com/groupon/vertx/utils/HealthcheckHandler.java]: checkstyle complained

File: src/main/java/com/groupon/vertx/utils/config/ConfigParser.java (hunk 2 above)
Comment (aleqi200) [reply src/main/java/com/groupon/vertx/utils/config/ConfigParser.java]: that was unintended ... weird I'll fix this in another PR

Comment (aleqi200) [reply src/main/java/com/groupon/vertx/utils/config/ConfigParser.java]: actually not, checkstyle too:
```
 RedundantModifier: Redundant 'final' modifier
```

**Pull request merged**